scripts/tests/test_validate_and_correct_eddy_gradients.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import tempfile
+
+from scilpy.io.fetcher import fetch_data, get_home, get_testing_files_dict
+
+# If they already exist, this only takes 5 seconds (check md5sum)
+fetch_data(get_testing_files_dict(), keys=['processing.zip'])
+tmp_dir = tempfile.TemporaryDirectory()
+
+
+def test_help_option(script_runner):
+    ret = script_runner.run('scil_validate_and_correct_eddy_gradients.py', '--help')
+    assert ret.success
+
+
+def test_execution_extract_half(script_runner):
+    os.chdir(os.path.expanduser(tmp_dir.name))
+    in_bvec = os.path.join(get_home(), 'processing', 'dwi.bvec')
+    in_bval = os.path.join(get_home(), 'processing', 'dwi.bval')
+    ret = script_runner.run('scil_validate_and_correct_eddy_gradients.py',
+                            in_bvec, in_bval, "32",
+                            'out.bvec',
+                            'out.bval')
```

Comment:
Aren't the tests shuffled at execution ? Add -f to this test case also if it ends up running before the one after

Reply:
Never seen it shuffled but I'm going to add the `-f`